src/GeneralStateTestsFiller/stEIP2930/manualCreateFiller.yml

```diff
@@ -0,0 +1,126 @@
+manualCreate:
+
+# Create contracts using contract creation transactions instead of 
+# CREATE or CREATE2 from the evm
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x20000'
+    currentGasLimit: '0xFF112233445566'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+
+
+  _info:
+    comment: Ori Pomerantz qbzzt1@gmail.com
+
+
+  pre:
+
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0x1000000000000000000
+      code: 0x
+      nonce: 1
+      storage: {}
+
+
+  transaction:
+    data:
+    # To create a contract directly, we send a transaction with the
```

Comment:
You can use lll code Return

Reply:
To provide contract code, but what's the point when I can't call it anyway (because after the transaction the call is over)? Or do you mean I should write a blockchain test for this?

Reply:
I mean you can write human readable code instead of bytecode